packages/bem-react-scripts/package.json

```diff
@@ -30,6 +30,7 @@
     "babel-loader": "7.0.0",
     "babel-plugin-bem-import": "1.3.2",
     "@babel/preset-env": "^7.0.0-beta.4",
+    "babel-preset-es2015": "^6.24.1",
```

Comment:
Can you also return [this](https://github.com/bem/create-bem-react-app/commit/a47681fec1bb32d554d20fa215d860b33bd4e6fc#diff-4fdcf69dd0b44749a3fc15d8ede40b58L67)

Reply:
Why we need two presets with the same functionality?

Reply:
because it is used to config https://github.com/bem/create-bem-react-app/blob/master/packages/bem-react-scripts/config/webpack.config.dev.js#L152

Reply:
Ok) We need only one ;)

Reply:
I think we need to remove @babel/preset-env

Reply:
fixed :)